libraries/botbuilder-ai/tests/luisRecognizer.test.js

```diff
@@ -486,6 +486,16 @@ describe('LuisRecognizer', function () {
         assert(traceActivity.value.luisModel);
     });
 
+    maybeIt('should not emit trace info if recognize called with utterance instead of TurnContext', async () => {
+        const expectedData = getExpectedData(getExpectedPath('SingleIntent_SimplyEntity.json'));
+
+        const res = await recognizer.recognize(expectedData.text);
+        assert(res);
+        assert(res.text === expectedData.text);
```

Comment:
For these, using `assert.strictEqual` tends to produce better (easier to debug) output.

Reply:
Fixed. I went ahead and also changed this throughout the whole test file because....I'm not sure why...I ask myself that every time.